src/app/state/staking/saga.test.ts

```diff
@@ -207,8 +251,8 @@ describe('Staking Sagas', () => {
               list: parseValidatorsList(
                 getMainnetDumpValidatorsMock.list.map(({ _expectedStatus, ...v }) => ({
```

Comment:
Test is passing, but I am not 100% sure about this. Test implements logic that is in saga?